File: update-types.js
const fs = require('fs');
const file = 'src/types/index.ts';
let content = fs.readFileSync(file, 'utf8');

content = content.replace(
  'codAvailable?: boolean;',
  'codAvailable?: boolean;\n  isPartialCOD?: boolean;\n  partialCODAdvance?: number;'
);

fs.writeFileSync(file, content, 'utf8');
console.log('types updated');

File: update_types.py
with open("src/types/index.ts", "r", encoding="utf-8") as f:
    content = f.read()

content = content.replace(
    "inventory?: { size?: string; stock: number; sku?: string }[];",
    "inventory?: { size?: string; stock: number; sku?: string }[];\n  lastEditedBy?: string;\n  updatedAt?: string;"
)

with open("src/types/index.ts", "w", encoding="utf-8") as f:
    f.write(content)
print("Updated Product type")
